service/k8sclient/k8client.go

```diff
@@ -295,6 +296,23 @@ func (c *K8Client) DeleteXtraDBCluster(ctx context.Context, name string) error {
 	return c.kubeCtl.Delete(ctx, res)
 }
 
+// RestartXtraDBCluster restarts Percona XtraDB cluster with provided name.
+func (c *K8Client) RestartXtraDBCluster(ctx context.Context, name string) error {
+	args := []string{"rollout", "restart", "StatefulSets", fmt.Sprintf("%s-pxc", name)}
```

Comment:
maybe we can extract generation of these args and calling kubectl.Run to separate function?
Because we use it in 3 places

Reply:
Extracted.